config/no-format-truncation.m4

```diff
@@ -0,0 +1,22 @@
+dnl #
```

Comment:
This file should be renamed `user-no-format-truncation.m4` since the configure check is only run as part of the user space build.